server/adaptors/integrations/__data__/repository/amazon_vpc_flow/amazon_vpc_flow-1.0.0.json

```diff
@@ -110,75 +98,12 @@
       "type": "savedObjectBundle",
       "workflows": ["flint-live-dashboards"]
     },
-    {
-      "name": "aws_vpc_flow_flint-pre_agg",
-      "version": "1.0.0",
-      "extension": "ndjson",
-      "type": "savedObjectBundle",
-      "workflows": ["flint-pre-agg-dashboards"]
-    },
-    {
-      "name": "vpc_live_all_mv",
+     {
+      "name": "aws_vpc_live_stream_mv_schema",
```

Comment:
Is this name format only for VPC? Should we also apply this on all the other integrations?